demos/testcurs.c

```diff
@@ -384,11 +381,7 @@ void scrollTest(WINDOW *win)
         wrefresh(win);
     };
 
-#ifdef PDCURSES
     OldY = getmaxy(win);
-#else
-    getmaxyx(win, OldY, OldX);
```

Comment:
@jwinarske How do you think about changing the old code to only use getmaxyx if neither PDCURSES nor NCURSES is defined? I *guess* the reason for getmaxyx was that the [standard curses definition][1] provides only this function.

I guess there may be some other missing ifdef's for a "standard-only" library to work with testcurs.c

[1];http://pubs.opengroup.org/onlinepubs/7908799/xcurses/curses.h.html

Reply:
Sounds like a good idea.  Ideally it would be best to have test cases to cover this kind of thing.

Reply:
Yes, test cases would be best, but as only xcurses has a configure script and testcurs.c should be usable with any installed curses version (that's actually the purpose - to be able to compare the installed version with the version you build) we cannot apply only a test case here.
But you're invited to add an additional `HAVE_GETMAXY` to the source and to the makefile descriptions if you want to and/or refactor the source to set this definition (if not defined already) in the head of the source file depending on other defines, this would be likely more clear than my quick edit.